task/build-vm-image/0.1/build-vm-image.yaml

```diff
@@ -200,9 +200,9 @@ spec:
 
         mkdir output
         echo "PULLING BUILDER IMAGE"
-        time sudo podman pull --authfile=$BUILD_DIR/.docker/config.json $BOOTC_BUILDER_IMAGE
+        time sudo podman pull --authfile=$BUILD_DIR/.docker/config.json --retry 10 $BOOTC_BUILDER_IMAGE
```

Comment:
Do I miss something here?
```
podman run --rm -ti quay.io/redhat-appstudio/multi-platform-runner:01c7670e81d5120347cf0ad13372742489985e5f@sha256:246adeaaba600e207131d63a7f706cffdcdc37d8f600c56187123ec62823ff44 podman pull --retry 10 fedora:40
Error: unknown flag: --retry
See 'podman pull --help'
```

Reply:
ah it's executed on remote host, what version of podman is installed there?

Reply:
Ah, so - that can vary depending on the deployment of MPC. The RH deployment of MPC uses RHEL 9 images, and RHEL 9 appstream (and ubi 9 appstream) includes podman-4:5.2.2-9.el9_5. I just confirmed: it does include the `--retry` option.